config/config.development.json

```diff
@@ -8,5 +8,9 @@
     "frontend": {
         "protocol": "http",
         "hostname": "localhost:8888"
-    }
+    },
+    "generate": {
+        "path": "./"
+    },
+    "upload": false
```

Comment:
This could be omitted, because `undefined` is still falsy, but better to be explicit.